json-lint/schemas/inventory/adservers/contentza.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$id": "#contentza",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Contentza adserver for TOUCH only",
+    "description": "Contentza ad server for TOUCH only",
+    "additionalProperties": false,
+    "required": [],
```

Comment:
Type must be required, otherwise the chunking mechanism will not work.

Reply:
done